governance/votes/Community Executive vote - January 22, 2021.md

```diff
@@ -0,0 +1,177 @@
+* The UNIV2WBTCETH-A vault type will be added using UNI-V2-WBTC-ETH as collateral with the parameters listed above.
+
+**[Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling) Increases**
+* The LINK-A Debt Ceiling will increase from 10 million DAI to 20 million DAI.
+* The MANA-A Debt Ceiling will increase from 250 thousand DAI to 1 million DAI.
+* The PSM-USDC-A Debt Ceiling will increase from 30 Million DAI to 80 Million DAI.
+* The PSM-USDC-A Debt Ceiling will increase linearly from 80 million DAI to 500 Million DAI over the next 12 weeks.
+
+**[Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling) Decreases**
+* The BAT-A Debt Ceiling will decrease from 10 million DAI to 2 million DAI.
+* The TUSD-A Debt Ceiling will decrease from 135 million DAI to zero DAI.
+* The Global System Debt Ceiling will decrease by 126,250,000 DAI.
+
+**[Debt Floor](https://community-development.makerdao.com/en/learn/governance/param-debt-floor) (`dust`) Changes**
+* The ETH-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The USDC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The WBTC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The ETH-B Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The TUSD-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The PAXUSD-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The LINK-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The COMP-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The YFI-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The UNI-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The UNIV2DAIETH-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The BAT-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The LRC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The BAL-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The renBTC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The AAVE-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The MANA-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The ZRX-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The KNC-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The GUSD-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The USDT-A Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+* The USDC-B Debt Floor (dust) parameter will increase from 500 DAI to 2000 DAI
+
+**DAI Distributions**
+* 6,300 Dai will be transferred from the Maker Protocol to @makerman at [0x9ac6a6b24bcd789fa59a175c0514f33255e1e6d0] for services rendered.
```

Comment:
Address is not in checksummed form. Should be 0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0